.gitignore

```diff
@@ -166,3 +166,5 @@ InnerEye-DataQuality/name_stats_scoring.png
 InnerEye-DataQuality/cifar-10-batches-py
 InnerEye-DataQuality/logs
 InnerEye-DataQuality/data
+
+!**/InnerEye/ML/Histopathology/datasets
```

Comment:
I think this is not needed, but will not hurt to have here. Maybe remove in next PR?